tests/test_global.py

```diff
@@ -0,0 +1,292 @@
+        username=nso_username,
+        password=nso_password,
+    )
+
+    filepath = georegion.execute_link(
+        "https://api.satellietdataportaal.nl/v1/download/30cm_RGBNED_12bit_PNEO/20230513_104139_PNEO-03_1_1"
+    )
+
+    assert filepath, "No file has been downloaded or extracted for multiPolygons"
+
+    # Remove the file for later tests
+    os.remove(filepath)
+
+
+# Channel adding tests.
+
+
+def test_re_ndvi_pneo():
+    # Test re_ndvi on PNEO coepelduynen and see if gives the best results.
+    path_geojson = "C:/repos/satellite-images-nso/input_data/Test_region.geojson"
+    output_path = "E:/data/test"
+
+    georegion = nso.nso_georegion(
+        path_to_geojson=path_geojson,
+        output_folder=output_path,
+        username=nso_username,
+        password=nso_password,
+    )
+
+    filepath = georegion.execute_link(
+        "https://api.satellietdataportaal.nl/v1/download/30cm_RGBNED_12bit_PNEO/20230513_104139_PNEO-03_1_1",
+        add_red_edge_ndvi_band=True,
+        plot=False,
+    )
+
+    # Open the TIFF image and display ndvi
+    with rasterio.open(filepath) as src:
+        # Get the number of bands
+        num_bands = src.count
+        print("Number of bands:", num_bands)
+
+        assert num_bands >= 7, "re_ndvi bands not added correctly"
+
+
+def test_adding_ndwi_pneo():
+    # Add the NDWI Channel
+    filepath_ndwi = nso_manipulator.add_index_channels(
+        "E:/data/test/20230513_104139_PNEO-03_1_1_30cm_RD_12bit_RGBNED_NoordwijkAanZee_Coepelduynen_cropped_Test_region_cropped_re_ndvi.tif",
+        ["ndwi"],
+    )
+
+    # Open the TIFF image and display ndvi
+    with rasterio.open(filepath_ndwi) as src:
+        # Get the number of bands
+        num_bands = src.count
+        print("Number of bands:", num_bands)
+
+        assert num_bands >= 8, "ndwi bands not added correctly"
+
+    os.remove(filepath_ndwi)
+
+
+filepath = ""
+
+
+def test_adding_ndvi_superview():
+    filepath = georegion.execute_link(
+        "https://api.satellietdataportaal.nl/v1/download/SV_RD_11bit_RGBI_50cm/20191202_110523_SV1-04",
+        add_ndvi_band=True,
+        plot=False,
+    )
+
+    # Open the TIFF image and display ndvi
+    with rasterio.open(filepath) as src:
+        # Get the number of bands
+        num_bands = src.count
+        print("Number of bands:", num_bands)
+
+        assert num_bands >= 5, "ndvi bands not added correctly"
+
+
+def test_adding_ndwi_superview():
+    # Add the NDWI Channel
+    filepath_ndwi = nso_manipulator.add_index_channels(
+        "E:/data/test/20191202_110523_SV1-04_50cm_RD_11bit_RGBI_NoordwijkAanZee_Coepelduynen_cropped_Test_region_cropped_ndvi.tif",
+        ["ndwi"],
+    )
+
+    # Open the TIFF image and display ndvi
+    with rasterio.open(filepath_ndwi) as src:
+        # Get the number of bands
+        num_bands = src.count
+        print("Number of bands:", num_bands)
+
+        assert num_bands >= 6, "re_ndvi bands not added correctly"
```

Comment:
I think this should read "re_ndwi bands not added correctly".

Reply:
corrected this